CONTRIBUTING.md

```diff
@@ -19,7 +19,7 @@ use GitHub pull requests for this purpose.
 
 All code submissions should try to match the surrounding code.  Wherever
 possible, code should adhere to the
-[Google Objective-C Style Guide](http://google-styleguide.googlecode.com/svn/trunk/objcguide.xml).
+[Google Objective-C Style Guide](https://google.github.io/styleguide/objcguide.xml).
```

Comment:
Looks like it moved again:
https://github.com/google/styleguide/blob/gh-pages/objcguide.md

Reply:
Ah, even better, that means there's now a HTML page to link to.